components/access-management/kfam/bindings.go

```diff
@@ -32,6 +32,15 @@ const ServiceRoleBinding = "servicerolebindings"
 const SERVICEROLEISTIO = "ns-access-istio"
 const USER = "user"
 const ROLE = "role"
+//roleBindingNameMap maps frontend role names to k8s role names and vice-versa
+var roleBindingNameMap = map[string]string {
+	"kubeflow-admin":	"admin",
```

Comment:
I guess we don't need mapping for both directions?

Reply:
@krishnadurai do we need both directions?

Reply:
@jlewi referring to the comment (here)[https://github.com/kubeflow/kubeflow/pull/4167#pullrequestreview-292398198].